app/models/provider.rb

```diff
@@ -69,6 +69,7 @@ def accredited_providers
   end
 
   alias accrediting_providers accredited_providers
+  alias accredited? accredited_provider?
```

Comment:
It is so confusing isn't it! `accredited_provider`, `accredited`, `accrediting_provider`, `ratifying_provider` 🤦🏼‍♀️

Reply:
I added this cause it's `accredited_provider.accredited_provider?` Redundancy. It should be obvious that we're calling `#accredited?` on a provider.